docs/FAQ.md

```diff
@@ -104,4 +105,17 @@ trusted certificates). This can be done in two ways:
   -Djavax.net.ssl.trustStore=NUL -Djavax.net.ssl.trustStoreType=Windows-ROOT
   ```
 
-For more information, see this [blog post](https://chancharles.medium.com/java-consultant-tip-ssl-certificates-and-man-in-the-middle-ssl-proxy-3867b81ee5f0).
\ No newline at end of file
+For more information, see this [blog post](https://chancharles.medium.com/java-consultant-tip-ssl-certificates-and-man-in-the-middle-ssl-proxy-3867b81ee5f0).
+
+### DelphiLint can't find my Java executable.
+
+Most Java installations set the `JAVA_HOME` environment variable, which points to the root directory of the Java
+installation. If this variable is present, DelphiLint defaults to `%JAVA_HOME%\bin\java.exe`.
+
+It's also possible to manually specify the path to the Java executable if `JAVA_HOME` isn't present, or points to the
+wrong Java version. This can be done in the DelphiLint External Resources Setup window at
+`DelphiLint > Settings > Set up external resources`.
+
+> [!NOTE]
+> Installers for unsupported old Java versions can sometimes not set `JAVA_HOME` - if you are having Java issues,
```

Comment:
I'd rather just have this be an outright **"AAAAAHHH PLEASE USE JAVA 11+"** message, as opposed to confusing the issue by bringing up `JAVA_HOME`.

Reply:
Actually, I think the "my java version is wrong" use-case could use its own section in here.

Reply:
Agreed.